toast-er.html

```diff
@@ -308,22 +308,26 @@
 
         // Get configuration of the toast.
         var bread = this._breads[0],
-          duration = bread.duration || this.duration,
-          type = bread.type || this.type,
+          buttonEvent = bread.buttonEvent || this.buttonEvent,
+          buttonText = bread.buttonText || this.buttonText,
+          hasButton = buttonText.length != 0,
+          duration = bread.duration || (hasButton ? 10000 : this.duration),
```

Comment:
@LM450N Well I think the developer should have total control over the element without being forced to have a value they don't want. I'll see what I can do. 

Maybe the default duration is the only thing that will be overridden, and if they specify a duration in the `iron-signal` it will take precedence. Something like: 
```js
duration = bread.duration || (hasButton && this.duration < 10000) ? 10000 : this.duration;
```